### ISSUE TITLE
Fix aria-required-children issue for axe upgrade

### ISSUE THREAD

**Issue by kerryarchibald** [vector-im/element-web#25342]
Changes to `aria-required-children` rule in a axe 4.7 cause failures with RoomList
https://github.com/matrix-org/matrix-react-sdk/pull/10645

Probably: https://github.com/dequelabs/axe-core/pull/3597

```
{
    id: 'aria-required-children',
    impact: 'critical',
    tags: [ 'cat.aria', 'wcag2a', 'wcag131' ],
    description: 'Ensures elements with an ARIA role that require child roles contain them',
    help: 'Certain ARIA roles must contain particular children',
    helpUrl: 'https://dequeuniversity.com/rules/axe/4.7/aria-required-children?application=axeAPI',
    nodes: [ [Object] ]
  }
  {
         "id":"aria-required-children",
         "data":{
            "messageKey":"unallowed",
            "values":"div[aria-label]"
         },
         "relatedNodes":[
            {
               "html":"<div class=\"mx_RoomSublist_headerContainer mx_RoomSublist_headerContainer_withAux\" aria-label=\"People\">",
               "target":[
                  "div[aria-label=\"People\"]"
               ]
            },
            {
               "html":"<div class=\"mx_RoomSublist_headerContainer mx_RoomSublist_headerContainer_withAux\" aria-label=\"Rooms\">",
               "target":[
                  ".mx_RoomSublist_headerContainer_withAux.mx_RoomSublist_headerContainer[aria-label=\"Rooms\"]"
               ]
            }
         ],
         "impact":"critical",
         "message":"Element has children which are not allowed: div[aria-label]"
      },
      ```
